x/concentrated-liquidity/swaps_tick_cross_test.go

```diff
@@ -21,6 +21,24 @@ func TestSwapTickCrossTestSuite(t *testing.T) {
 	suite.Run(t, new(SwapTickCrossTestSuite))
 }
 
+const (
+	tickSpacingOne = 1
+	tickSpacing100 = 100
+)
+
+// This configuratio is expected to generate the position layout below:
+//
+//	                                 original_cur_tick
+//		                                    ///               (NR4)
+//		                                /////////             (NR3)
+//		                            ////////////////          (NR2)
+//	                             //////////////////////       (NR1)
```

Comment:
I think indentation got  messed up here :L(

Reply:
Hmm, weird. It looks fine for me locally. If I change it, it gets linted back to this state.

Are yo OK with not blocking on this? I think this still gives a good idea of the layout